plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/search_investigations/schema.py

```diff
@@ -111,28 +111,9 @@ class SearchInvestigationsOutput(insightconnect_plugin_runtime.Output):
     }
   },
   "required": [
-    "investigations",
```

Comment:
was this meant to be removed 

```
      output:
        investigations:
          title: Investigations
          description: A list of found investigations
          type: "[]investigation"
          required: true
```

as from the plugin spec this looked to be required true